### DISCUSSION TITLE
Milvus 0.10.2 -Whether the merging of two collections is supported

### DISCUSSION THREAD

**Discussion by Haohao966** [milvus-io/milvus#3789] [Q&A and General discussion]
I want to merge the data from collection2 into collection1 to make a large collection,
Whether this operation is supported

**Comment by shiyu22** [comment]
Currently, we don't support this, and can you tell me what the scenario is where you need this to operate?

If necessary, please submit an [issue](https://github.com/milvus-io/milvus/issues) to our project.
